src/lobby.cpp

```diff
@@ -524,6 +526,9 @@ void Lobby::list_servers()
 
 void Lobby::list_favorites()
 {
+  if (public_servers_selected) {
+    return;
+  }
```

Comment:
```suggestion
```
This check is redundant as the only way to open the favorite menu is to use the UI element, meaning it will never auto-open, unlike the serverlist, which is opened when the client receives the serverlist from ms3.